src/librustdoc/visit_ast.rs

```diff
@@ -213,8 +215,36 @@ impl<'a, 'tcx> RustdocVisitor<'a, 'tcx> {
 
     }
 
-    fn resolve_id(&mut self, id: ast::NodeId, renamed: Option<ast::Name>,
+    /// Tries to resolve the target of a `pub use` statement and inlines the
```

Comment:
ah yeah private imports probably never want to be inlining as it's just bringing a name into scope